web/template/partial/course/manage/create-lecture-form.gohtml

```diff
@@ -15,6 +15,11 @@
                     <input type="checkbox" :disabled="formData.premiere || formData.vodup" x-model="formData.recurring"
                            class="w-auto">
                     <span class="text-3">Recurring Event</span>
+                </label><br>
+                <label>
+                    <input type="checkbox" x-model="formData.isChatEnabled" x-init="formData.isChatEnabled = {{.IndexData.TUMLiveContext.Course.ChatEnabled}}"
```

Comment:
Can we hide/gray this out when the chat is disabled globally? I think you can just use `{{.IndexData.TUMLiveContext.Course.ChatEnabled}}` and then track whether the user disabled/enabled until next reload.

Reply:
Good idea! Now, it is implemented.